File: ARCADE/Intro/Level 1_1: add.cs
using System;

public class Test
{
	public static int add(int param1, int param2){
		return param1 + param2;
	}
	public static void Main()
	{
		int param1 = 1, param2 = 2;
		Console.WriteLine(add(param1,param2));
	}
}

File: ARCADE/Intro/Level 1_1: add.pas
program add;

function add(param1: integer; param2: integer): integer;
begin
  add := param1 + param2;
end;

var param1, param2, result : integer;
begin
	param1 := 1; param2 := 2;
	writeln(add(param1, param2));
end.
